templates/caddy.service

```diff
@@ -68,6 +68,15 @@ NoNewPrivileges=true
 Environment={{ key }}={{ value }}
 {% endfor %}
 
+{% endif %}
+{% if caddy_environment_files|length %}
+
+; Additional environment files:
+
+{% for path in caddy_environment_files %}
+EnvironmentFile=-{{ path }}
```

Comment:
Could you remove the dash? I would prefer if errors/warnings were enabled by default and the user can add them to the path directly if they prefer this behavior.

Reply:
Sure